include/inout.h

```diff
@@ -65,6 +65,11 @@ namespace graft
         {
             return std::make_pair(m_buf.c_str(), m_buf.length());
         }
+
+        std::string data() const
```

Comment:
do you mean `std::string get() const;`?
How it possible? The compiler will throw an error since it cannot choose a right implementation of a method.
Or you mean something else?